### PR TITLE
Adding missing emojis and correcting typo

### PR DESCRIPTION
**Pull request by mohitm15**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

### Description

Adding missing emojis in styleguide.md and correcting the typo in translations.md

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: docs/styleguide.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ Where possible, [automated tests](../test/prose) enforce style rules.
 ## Content principles
 All written content should follow these principles:
 
-* **Approachability:** Don't assume reader has prior knowledge
-* **Brevity:** Keep it simple, link to outside content for deeper dives
-* **Curation:** Amplify community best practices vs. any individual's point of view
+* :smile: **Approachability:** Don't assume reader has prior knowledge
+* :smile: **Brevity:** Keep it simple, link to outside content for deeper dives
+* :cry: **Curation:** Amplify community best practices vs. any individual's point of view
```
Comment (MikeMcQuaid) [inline docs/styleguide.md]: Can you remove these? Thanks!

File: docs/translations.md [hunk 2]
```diff
@@ -34,4 +34,4 @@ We're collecting [tips](https://github.com/github/opensource.guide/issues/1119)
 
 ### New articles
 
-New articles are rare! When we have one, we'll probably do [some form](https://github.com/github/opensource.guide/issues/1120) of a call for translations.
\ No newline at end of file
+New articles are rare! When we have one, we'll probably do [some form](https://github.com/github/opensource.guide/issues/1120) of a call for translations.
```
Comment (MikeMcQuaid) [inline docs/translations.md]: What changed here?

File: docs/translations.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ If there's not, then today is your day to lead this effort! Here's how to start:
 
 Completing an initial translation of the whole site is a fairly large task. One way to break that task up is to work with other translators through pull requests on your fork. Example: [pull requests on fork for German translation](https://github.com/katrinleinweber/opensource.guide/pulls?q=is%3Apr+is%3Aclosed) and corresponding [initial pull request for German translation](https://github.com/github/opensource.guide/pull/577) on this repository.
 
-## Updatng a translation
+## Updating a translation
```
Comment (MikeMcQuaid) [inline docs/translations.md]: 👍

File: docs/styleguide.md [hunk 4]
```diff
@@ -39,4 +39,4 @@ When referring to a project on GitHub, link to the repository so others can dive
 
 ## More guidance
 
-Understand our [content model](content-model.md) and [audience](personas.md)
\ No newline at end of file
+Understand our [content model](content-model.md) and [audience](personas.md)
```
Comment (MikeMcQuaid) [inline docs/styleguide.md]: What changed here?

File: docs/translations.md (hunk 2 above)
Comment (mohitm15) [reply docs/translations.md]: I didn't made any change here, I think there may be a shift in space

File: docs/styleguide.md (hunk 4 above)
Comment (mohitm15) [reply docs/styleguide.md]: I didn't made any change here, I think there may be a shift in space

File: docs/styleguide.md (hunk 1 above)
Comment (mohitm15) [reply docs/styleguide.md]: Sure sir

**Comment by mohitm15** [comment]
@MikeMcQuaid should i open another pR for second change of typo

File: docs/styleguide.md (hunk 4 above)
Comment (MikeMcQuaid) [reply docs/styleguide.md]: Ok, np!

File: docs/translations.md (hunk 2 above)
Comment (MikeMcQuaid) [reply docs/translations.md]: Ok, np!

**Comment by MikeMcQuaid** [comment]
@mohitm15 Please push another commit to this PR instead. Thanks!

**Comment by mohitm15** [comment]
> @mohitm15 Please push another commit to this PR instead. Thanks!

Please review the pr now @MikeMcQuaid

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @mohitm15!

**Comment by mohitm15** [comment]
> Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @mohitm15!

your welcome @MikeMcQuaid

**Pull request merged**